src/html5printshiv.js

```diff
@@ -31,7 +30,7 @@ define(['isSVG'], function(isSVG) {
       var expando = '_html5shiv';
 
       /** The id for the the documents expando */
-      var expandoID = 0;
+      var expanID = 0;
```

Comment:
this is not our code. don't change any of this file.

Reply:
I know. But I didnt know back when I asked you to merge https://github.com/Modernizr/Modernizr/pull/2267/ where the foreign file got "spellchecked" :-) Just undoing my mistake from back then